doc/develop/test/bsim.rst

```diff
@@ -0,0 +1,72 @@
+.. _bsim:
+
+BabbleSim
+#########
+
+BabbleSim and Zephyr
+********************
+
+In the Zephyr project we use the `Babblesim`_ simulator to test some of the Zephyr radio protocols,
+including the BLE stack, 802.15.4, and some of the networking stack.
+
+BabbleSim_ is a physical layer simulator, which in combination with the Zephyr
+:ref:`bsim boards<bsim boards>`
+can be used to simulate a network of BLE and 15.4 devices.
+When we build Zephyr targeting an :ref:`nrf52_bsim<nrf52_bsim>` board we produce a Linux
+executable, which includes the application, Zephyr OS, and models of the HW.
+
+When there is radio activity, this Linux executable will connect to the BabbleSim Phy simulation
+to simulate the radio channel.
+
+In the BabbleSim documentation you can find more information on how to
+`get <https://babblesim.github.io/fetching.html>`_. and
```

Comment:
Does it render correctly by just using \` or do you need to add `:code:` as we do many other places?

Reply:
It renders fine. There is just too many ways of adding links in rst :) . I can add a :ref: if it feels nicer.

Reply:
Sorry, this was a bad example :) 

Let's take the 
```
Each test is defined by a shell script with the extension `.sh`.
```
as an example instead; does the `.sh` render properly, or does it does show as \`.sh\`.?

Reply:
That particular example is the README which is markdown :) (I converted the .txt to .md to add a tiny bit of format)